.eslintrc.cjs

```
@@ -0,0 +1,98 @@
+        'jsx-a11y/no-interactive-element-to-noninteractive-role': 'warn',
+        'jsx-a11y/no-noninteractive-element-interactions': 'warn',
+        'jsx-a11y/no-noninteractive-element-to-interactive-role': 'warn',
+        'jsx-a11y/no-noninteractive-tabindex': 'warn',
+        'jsx-a11y/no-redundant-roles': 'warn',
+        'jsx-a11y/no-static-element-interactions': 'warn',
+        'jsx-a11y/role-has-required-aria-props': 'warn',
+        'jsx-a11y/role-supports-aria-props': 'warn',
+        'jsx-a11y/scope': 'warn',
+        'jsx-a11y/tabindex-no-positive': 'warn'
+    }
+};
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)